src/lib/cfa/nm_connection.rb

```diff
@@ -22,28 +22,68 @@
 module CFA
   # Class to handle NetworkManager connection configuration files
   #
+  # @see https://developer.gnome.org/NetworkManager/stable/nm-settings-keyfile.html
   # @example Reading the connection name
   #   file = NmConnection.new("/etc/NetworkManager/system-connections/eth0.nmconnection")
   #   file.load
   #   puts file.connection["id"]
   class NmConnection < BaseModel
+    KNOWN_SECTIONS = [
+      "bridge", "connection", "ethernet", "ipv4", "ipv6", "vlan", "wifi", "wifi_security"
+    ].freeze
+
+    # @return [String] file path
+    attr_reader :path
+
     # Constructor
     #
     # @param path [String] File path
     # @param file_handler [.read, .write] Object to read/write the file.
     def initialize(path, file_handler: nil)
-      super(AugeasParser.new("Networkmanager.lns"), path, file_handler: file_handler)
+      @path = path
+      # FIXME: The Networkmanager lense writes the values surrounded by double
+      # quotes which is not valid
```

Comment:
was it reported as a bug to the augeas maintainer?

Reply:
Not yet, is it in my TODO list. Do you want to take care? that would be nice